### PR TITLE
Made changes in the MySamples Page for Marshfield_Issue

### PR DESCRIPTION
**Pull request by ramavavi**
The following Pull request (PR) addresses: https://github.com/episphere/connectApp/issues/548

Updates to the code:

1. Made changes to Locations and the timings.
2. Made a few edits under the heading How do I donate my samples?

### CODE REVIEW

File: js/pages/samples.js [hunk 1]
```diff
@@ -328,7 +328,7 @@ const marshfield = {
...
-    howToDonate: 'When you receive the email, please call Marshfield Clinic Research Institute at 715-898-9444 to schedule an appointment to donate your samples at a time that is convenient for you.',
+    howToDonate: 'The email we send you will contain a link to schedule an appointment. Simply click the link to schedule a time that is convenient for you to donate your samples.You can also  please call Marshfield Clinic Research Institute at 715-898-9444 to schedule an appointment, or a Connect team member will call you to schedule an appointment to donate your samples at a time that is convenient for you.',
```
Comment (Gbarra9) [inline js/pages/samples.js:331]: The email we send you will contain a link to schedule an appointment. Simply click the link to schedule a time that is convenient for you to donate your samples.You can also please call Marshfield Clinic Research Institute at 715-898-9444 to schedule an appointment, or a Connect team member will call you to schedule an appointment to donate your samples at a time that is convenient for you.
```suggestion
    howToDonate: 'The email we send you will contain a link to schedule an appointment. Simply click the link to schedule a time that is convenient for you to donate your samples.You can also call Marshfield Clinic Research Institute at 715-898-9444 to schedule an appointment, or a Connect team member will call you to schedule an appointment to donate your samples at a time that is convenient for you.',
```

I think from the [document]([Marshfield_Biospecimen Info Template for PWA_UpdatesMay2023.docx](https://github.com/episphere/connectApp/files/11432673/Marshfield_Biospecimen.Info.Template.for.PWA_UpdatesMay2023.docx)) the word "please" is removed.

File: js/pages/samples.js [hunk 2]
```diff
@@ -338,20 +338,28 @@ const marshfield = {
         [
             'Marshfield Clinic, Marshfield Center',
             '1000 N. Oak Ave<br>Marshfield, WI 54449',
-            'Monday - Friday: 8:00 a.m.-2:00 p.m. *please ask about early morning or evening appointment availability if needed',
+            'Monday - Thursday: 7:00 a.m. - 4:00 p.m. and Friday 7a.m. - noon',
```
Comment (Gbarra9) [inline js/pages/samples.js:341]: Can you confirm this is the correct text? From the document I see `Monday – Thursday: 7:00 a.m. to 4:00 p.m. and Friday 7:00 a.m. to noon.` This is also the case for this [line 347](https://github.com/episphere/connectApp/pull/565/files#diff-086e163ff162b94330737cfb586d27c44a4871d7792612bf21d2791ead9d7c29R347)

File: js/pages/samples.js [hunk 3]
```diff
@@ -328,7 +328,7 @@ const marshfield = {
...
-    howToDonate: 'The email we send you will contain a link to schedule an appointment. Simply click the link to schedule a time that is convenient for you to donate your samples.You can also  please call Marshfield Clinic Research Institute at 715-898-9444 to schedule an appointment, or a Connect team member will call you to schedule an appointment to donate your samples at a time that is convenient for you.',
+    howToDonate: 'The email we send you will contain a link to schedule an appointment. Simply click the link to schedule a time that is convenient for you to donate your samples.You can also call Marshfield Clinic Research Institute at 715-898-9444 to schedule an appointment, or a Connect team member will call you to schedule an appointment to donate your samples at a time that is convenient for you.',
```
Comment (JoeArmani) [inline js/pages/samples.js:331]: It looks like we need a space between 'samples.' and 'You can'.
...`convenient for you to donate your samples. You can also call Marshfield`...

**Review by JoeArmani — CHANGES_REQUESTED** [review]

File: js/pages/samples.js (hunk 3 above)
Comment (JoeArmani) [reply js/pages/samples.js]: Looks good on this change, marking this resolved.

File: js/pages/samples.js [hunk 4]
```diff
@@ -338,20 +338,28 @@ const marshfield = {
...
+            'Mondays and Thursdays: 8:00 a.m. - 1:00 p.m. *please ask about evening and Friday appointment availability if needed',
             'General parking available'
+        ],
+        [
+            'Wisconsin Rapids Center',
+            '220 24th St S,<br>Wisconsin Rapids,WI 54494',
+            'Thursdays: 7:00 a.m. - 4:00 p.m. <br> Fridays: 7:00 a.m. - 10:00 a.m.',
+            'General parking available'
+
+
```
Comment (Gbarra9) [inline js/pages/samples.js:362]: Remove extra lines on line 361 and line 362.

**Review by Gbarra9 — COMMENTED** [review]
Looks good. I added a minor suggestion.

**Review by abhinavjonnada82 — APPROVED** [review]

**Review by Gbarra9 — APPROVED** [review]
Looks good

**Review by anthonypetersen — APPROVED** [review]
looks good

**Review by JoeArmani — APPROVED** [review]
Looks good

**Pull request merged**